gatsby/content/projects/2018-12-27-fluffychat.mdx

```diff
@@ -4,40 +4,43 @@ title: FluffyChat
 categories: 
  - client
 thumbnail: https://christianpauly.gitlab.io/fluffychat-website/assets/images/screenshot.png
-description: Open, nonprofit and cute client for Ubuntu Touch
+description: Chat with your friends
 author: Christian Pauly
 maturity: Released
-language: QML
+language: Flutter
 license: GPL-3.0-only
```

Comment:
The license of the new project seems to be AGPL-3.0-only instead.

Reply:
Oh you are right. Thank you!